Source/ExcelDna.sln

```diff
@@ -25,6 +25,16 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ExcelDna.IntegrationTests",
 		{A8D668E2-63C1-4F4E-A01E-E026C6E4D601} = {A8D668E2-63C1-4F4E-A01E-E026C6E4D601}
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Package", "Package", "{2A988F4A-6C24-49EB-950D-0984E2CAEE20}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ExcelDna.AddIn", "ExcelDna.AddIn", "{42A3C1DE-5D65-4BC6-BA90-3FDEE42A77C9}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "content", "content", "{D0B02D05-F536-4C32-9DC0-BA2B5EEAEAB7}"
+	ProjectSection(SolutionItems) = preProject
+		..\Package\ExcelDna.AddIn\content\ExcelDna-Template.dna = ..\Package\ExcelDna.AddIn\content\ExcelDna-Template.dna
+		..\Package\ExcelDna.AddIn\content\ExcelDna_dna_2018_05.xsd = ..\Package\ExcelDna.AddIn\content\ExcelDna_dna_2018_05.xsd
```

Comment:
Not sure what if this file still exists.

Reply:
Yep, good catch, this line needs to go.